src/yunohost/app.py

```diff
@@ -483,6 +512,9 @@ def app_upgrade(app=[], url=None, file=None):
         env_dict["YNH_APP_ID"] = app_id
         env_dict["YNH_APP_INSTANCE_NAME"] = app_instance_name
         env_dict["YNH_APP_INSTANCE_NUMBER"] = str(app_instance_nb)
+        env_dict["YNH_APP_UPGRADE_TYPE"] = upgrade_type
+        env_dict["YNH_APP_MANIFEST_VERSION"] = app_new_version
+        env_dict["YNH_APP_OLD_VERSION"] = app_actual_version
```

Comment:
```suggestion
        env_dict["YNH_APP_CURRENT_VERSION"] = app_actual_version
```
OLD isn't really clear about what we're talking about. Stumble on it on the helper above.